README.md

```diff
@@ -157,5 +157,5 @@ Additional documentation is found [here](/docs/tools).
 
 - [The latest gossip on BFT consensus](https://arxiv.org/abs/1807.04938)
 - [Master's Thesis on Tendermint](https://atrium.lib.uoguelph.ca/xmlui/handle/10214/9769)
-- [Original Whitepaper](https://tendermint.com/static/docs/tendermint.pdf)
+- [Original Whitepaper](https://cosmos.network/resources/whitepaper)
```

Comment:
```suggestion
- [Original Whitepaper](https://github.com/tendermint/spec)
   - You can find the link at the bottom of the readme
```

Thank you for your contribution and for finding this broken link. The paper to which it is referring to located here: https://github.com/tendermint/spec. The cosmos whitepaper does not have relevance here. You can put something along the lines as the above suggestion.